mm2src/mm2_main/tests/docker_tests/docker_tests_inner.rs

```diff
@@ -5517,3 +5517,84 @@ fn test_orderbook_depth() {
     block_on(mm_bob.stop()).unwrap();
     block_on(mm_alice.stop()).unwrap();
 }
+
+#[test]
+fn test_approve_erc20() {
+    let privkey = random_secp256k1_secret();
+    fill_eth_erc20_with_private_key(privkey);
+
+    let coins = json!([eth_dev_conf(), erc20_dev_conf(&erc20_contract_checksum())]);
+    let mm = MarketMakerIt::start(
+        json!({
+            "gui": "nogui",
+            "netid": 9000,
+            "dht": "on",  // Enable DHT without delay.
+            "passphrase": format!("0x{}", hex::encode(privkey)),
+            "coins": coins,
+            "rpc_password": "pass",
+            "i_am_seed": true,
+        }),
+        "pass".to_string(),
+        None,
+    )
+    .unwrap();
```

Comment:
better to use this method https://github.com/KomodoPlatform/komodo-defi-framework/blob/26b87c3949d324f33b0f7f881721f2107d82c725/mm2src/mm2_test_helpers/src/for_tests.rs#L257-L269 for conf. Also, lightnode is enough for this test.

Reply:
used seednode
(apparently lightnode does not start alone)